.github/workflows/openapi.yaml

```
@@ -21,11 +31,30 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
 
-      - name: Validate open-api-spec.yaml
+      - name: AsyncAPI extension
         run: |
           echo "{\"extends\":[\"spectral:oas\",\"spectral:asyncapi\"]}" >> .spectral.json
+
+      # TODO: remove old ones
+      - name: Validate open-api-spec.yaml
+        run: |
           npx @stoplight/spectral-cli lint open-api-spec.yaml
```

```suggestion
          npx @stoplight/spectral-cli lint ./openapi-specs/**/openapi.yaml
```
https://meta.stoplight.io/docs/spectral/9ffa04e052cc1-spectral-cli

---

Are you suggesting doing that instead of having them all individually?

---

Yeah